### PR TITLE
Update documentation for snmp nodes

### PR DESCRIPTION
**Pull request by bimalyn-IBM**
I've updated the README as well as the html file to help explain the changes made, which allow users to specify the host and community for the snmp nodes by setting `msg.host` and `msg.community` respectively.

### CODE REVIEW

File: io/snmp/README.md [hunk 1]
```diff
@@ -18,9 +18,17 @@ Usage
 
 SNMP oids fetcher. Can fetch a single or comma separated list of oids. Triggered by any input.
 
+`msg.host` may contain the host, iff host not set in the node configuration.
```
Comment (dceejay) [inline io/snmp/README.md]: sp ?  iff -> if

File: io/snmp/README.md [hunk 2]
```diff
@@ -18,9 +18,17 @@ Usage
...
+
+`msg.community` may contain the community, iff community not set in the node configuration.
```
Comment (dceejay) [inline io/snmp/README.md]: iff ?

File: io/snmp/README.md [hunk 3]
```diff
@@ -18,9 +18,17 @@ Usage
...
+
 `msg.oid` may contain a comma separated list of oids to search for. (no spaces)
 
-The oids confgured in the edit config will override `msg.oid`. Leave blank if you
+The host configured in the edit config will conflict with `msg.host`. Leave blank if you want to use `msg.host` to provide input.
```
Comment (dceejay) [inline io/snmp/README.md]: it doesn't conflict - it's a choice. just need the second sentence.

File: io/snmp/README.md [hunk 4]
```diff
@@ -18,9 +18,17 @@ Usage
...
+
+The community configured in the edit config will conflict with `msg.community`. Leave blank if you want to use `msg.community` to provide input.
```
Comment (dceejay) [inline io/snmp/README.md]: again - no need to say it conflicts.

File: io/snmp/README.md [hunk 5]
```diff
@@ -18,9 +18,17 @@ Usage
...
+
+The oids configured in the edit config will override `msg.oid`. Leave blank if you
```
Comment (dceejay) [inline io/snmp/README.md]: Correct !

File: io/snmp/README.md [hunk 6]
```diff
@@ -29,8 +37,37 @@ Values depends on the oids being requested.
 ### snmp-table
 
 Simple SNMP table oid fetcher. Triggered by any input.
+
```
Comment (dceejay) [inline io/snmp/README.md]: Same comments apply

File: io/snmp/README.md [hunk 7]
```diff
@@ -39,4 +76,20 @@ Values depends on the oids being requested.
 
 ### snmp-walker
 
-### snmp-subtree
+Simple SNMP oid walker fetcher. Triggered by any input.
```
Comment (dceejay) [inline io/snmp/README.md]: Likewise again ... iff ?  no conflicts - overrides is OK.

**Comment by bimalyn-IBM** [comment]
Sorry I'll change around the default behaviour to copy this style:
```
var oids = node.oids || msg.oid;
```
and modify my comments to match (and fixing the iffs)

**Pull request merged**